### PR TITLE
Refactor code for packaging

### LINKED ISSUE #15: Make Bassa Python Client Library v1.0.0-alpha.1 release
To make the release we need to set up a pipeline to the Pypi for publishing the client library as a python package.

### PR DESCRIPTION
**Pull request by kmehant**

### CODE REVIEW

**Comment by kmehant** [comment]
@JaDogg Please Review

File: python_lib/setup.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from setuptools import setup
+
+def readme():
+    with open('README.md') as f:
+        README = f.read()
+    return README
+
+setup(
+
```
Comment (JaDogg) [inline python_lib/setup.py:9]: Question: Is this suppose to be empty?

**Review by JaDogg — APPROVED** [review]

File: python_lib/setup.py (hunk 1 above)
Comment (kmehant) [reply python_lib/setup.py:9]: I have @agentmilindu regarding the author details. So need to fill this up 😁

**Comment by kmehant** [comment]
@JaDogg 

Will merge this PR. I will complete `setup.py` once I will get a response from @agentmilindu bro regarding the author credentials and ownership.

File: python_lib/setup.py (hunk 1 above)
Comment (JaDogg) [reply python_lib/setup.py:9]: ah good point

**Pull request merged**